apps/web/src/pages/CommunityInterests/CommunityTalentPage/components/CommunityTalentFilterDialog.tsx

```diff
@@ -0,0 +1,293 @@
+                value: "trainingInterest",
+              },
+            ]}
+          />
+          <Checklist
+            idPrefix="mobilityType"
+            name="mobilityType"
+            legend={intl.formatMessage({
+              defaultMessage: "Mobility type",
+              id: "Avr7yn",
+              description:
+                "Legend for mobility type checklist in community talent filter dialog",
+            })}
+            items={[
+              {
+                label: intl.formatMessage({
+                  defaultMessage:
+                    "Interested in Lateral movement (At-level mobility)",
+                  id: "+CU+6M",
+                  description:
+                    "Label for interested in lateral movement checkbox for mobility type checklist",
+                }),
+                value: "lateralMoveInterest",
+              },
+              {
+                label: intl.formatMessage({
+                  defaultMessage: "Promotions and advancement",
+                  id: "h0mWc3",
+                  description:
+                    "Label for interested in promotional movement checkbox for mobility type checklist",
+                }),
+                value: "promotionMoveInterest",
+              },
+            ]}
+          />
+        </div>
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x1 0)"
+        >
+          <Select
+            id="languageAbility"
+            name="languageAbility"
+            enableNull
+            nullSelection={intl.formatMessage(commonMessages.anyLanguage)}
+            label={intl.formatMessage(commonMessages.workingLanguageAbility)}
+            options={languageAbilities.map(({ value, label }) => ({
+              value,
+              label: label.localized,
+            }))}
+          />
+          <Select
+            id="employmentDuration"
+            name="employmentDuration"
+            enableNull
+            nullSelection={intl.formatMessage({
+              defaultMessage: "Any duration",
+              id: "Swq+OD",
+              description: "Option label for allowing any employment duration",
+            })}
+            label={intl.formatMessage({
+              defaultMessage: "Duration preferences",
+              id: "2ingb6",
+              description: "Label for the employment duration field",
+            })}
+            options={enumToOptions(EmploymentDuration).map(({ value }) => ({
+              value,
+              label: intl.formatMessage(getEmploymentDuration(value, "short")),
+            }))}
+          />
+        </div>
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x1 0)"
+        >
+          <Checklist
+            idPrefix="workRegions"
+            name="workRegions"
+            legend={intl.formatMessage(navigationMessages.workLocation)}
```

Comment:
Should this be plural "Work locations" since it is a multi combo box?

Reply:
This isn't a multi combo box. Also, this one is consistent with the other filter dialogs that also share this filter (e.g. UserFilterDialog). To change the text of this would require changing all the other filters as well. We should probably create a new issue for that. 

Also, since it's a checklist they can select zero, one, or more than one option. The label would need to be Work location(s).

Reply:
Whoops, copy and paste is not my friend: it is not a multi combo box!

> To change the text of this would require changing all the other filters as well. We should probably create a new issue for that.

👍